content/en/docs/setup/best-practices/certificates.md

```diff
@@ -57,7 +58,18 @@ Required CAs:
 | front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/) |
 
 On top of the above CAs, it is also necessary to get a public/private key pair for service account management, `sa.key` and `sa.pub`.
+The following example illustrates the CA key and certificate files shown in the previous table:
 
+```shell
```

Comment:
This is not a valid shell script. Did you mean <tt>```</tt>?

Reply:
I just wanted it to print out as a block of monospaced text. I can change it to just backticks if that works.

Reply:
Just a nit, but please do

Reply:
Done.